content/events/2020-guadalajara/location.md

```diff
@@ -0,0 +1,10 @@
++++
+Title = "Location"
+Type = "event"
+Description = "Location for devopsdays Guadalajara 2020"
++++
+
+Watch this space for information about the venue including address, map/direction, parking/transit, and any hotel details.
+
+<!-- Uncomment this only if you have set the coordinates for your location in the config yaml. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html -->
+<!-- {{< event_map >}} -->
```

Comment:
Since we'll set the `event_map`, let's uncomment this section

Reply:
ok, link is now available on the site